<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScheduleTab</class>
 <widget class="QWidget" name="ScheduleTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QTableWidget" name="trainingTable">
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="columnCount">
      <number>6</number>
     </property>
     <column>
      <property name="text">
       <string>Category</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Provider</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Date Completed</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Renewal Period (Months)</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Expiry Date</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Tools</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="2">
       <widget class="QPushButton" name="saveTrainingButton">
        <property name="text">
         <string>Save</string>
        </property>
        <property name="icon">
         <iconset theme="Save">
          <normaloff>.</normaloff>.</iconset>
        </property>
        <property name="shortcut">
         <string>Ctrl+S</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QComboBox" name="nameSelectorBox">
        <property name="currentText">
         <string>All Categories</string>
        </property>
        <item>
         <property name="text">
          <string extracomment="Show all records">All Categories</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QPushButton" name="deleteTrainingButton">
        <property name="text">
         <string>Delete Rows</string>
        </property>
        <property name="shortcut">
         <string>Del</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="addTrainingButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Add New Item</string>
        </property>
        <property name="shortcut">
         <string>Ctrl+N</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QPushButton" name="printButton">
        <property name="text">
         <string>Print</string>
        </property>
        <property name="shortcut">
         <string>Ctrl+P</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
